earth_enterprise/src/SConstruct

```diff
@@ -480,6 +480,10 @@ gee_version_number = env.get_open_gee_version().get_short()
 
 
 # Check if the version of the default compiler is at least 4.8:
+if not env['CXX']:
+    print "Please, install the default C++ compiler for your distribution."
+    Exit(1)
+
 version = opengee.c_compiler.get_cc_version(env['CXX'])
```

Comment:
Couldn't you also fix this by first checking to see if env['CXX'] is set? If it isn't, then you can still do the check for devtools below.

Reply:
The system default compiler is required for building the third party modules.  Without it, the build fails here:

```
checking for c++... /home/jeff/earthenterprise/earth_enterprise/src/NATIVE-REL-x86_64/third_party/bin/g++
checking whether the C++ compiler (/home/jeff/earthenterprise/earth_enterprise/src/NATIVE-REL-x86_64/third_party/bin/g++  ) works... no
configure: error: installation or configuration problem: C++ compiler cannot create executables.
```

This is because `third_party/bin/g++` is created with a hardcoded reference to `/usr/bin/g++`, even after running `source /opt/rh/devtoolset-2/enable`.

There might need to be a new issue to allow the third party modules to be built with another C++ compiler (e.g. the devtoolset toolchain).

Reply:
Ah, that makes sense. I'll write a follow-up ticket to fix that issue.

Reply:
Added #1142